src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
-// write code here
+const addRowBtn = document.querySelector('.append-row');
+const addColumnBtn = document.querySelector('.append-column');
+const deleteRowBtn = document.querySelector('.remove-row ');
+const deleteColumnBtn = document.querySelector('.remove-column');
+
+const table = document.querySelector('.field');
+
+const maxCount = 10;
+const minCount = 2;
+let rowCount = table.rows.length;
```

Comment:
The initial value of 'rowCount' is set only once when the script is loaded. This might cause problems if the number of rows in the table changes after the script is loaded. You should consider updating 'rowCount' every time the table is changed.